src/bootstrap.php

```diff
@@ -454,6 +454,9 @@
     }
 
     foreach ($reports as $i => $report) {
+        if ($type->getParameter('version') < 2 && 2014 === $report['year']) {
```

Comment:
It'd be better to test for the presence of `$report['image']`, but instead let's keep the data, and instead `unset()` the image when requesting version 2. We can remove the image when we drop version 1.